src/javascript/app_2/App/Components/Form/DatePicker/date-picker.jsx

```diff
@@ -230,7 +230,7 @@ class DatePicker extends React.Component {
             >
                 { this.renderInputField() }
                 <IconCalendar
-                    className={classNames('datepicker__icon datepicker__icon--calendar', {
+                    className={classNames('input-field__icon datepicker__icon datepicker__icon--calendar', {
```

Comment:
`input-field__icon `?

Reply:
these icons that project inside the input field have common styling. I can rename it and move it out of input-field.scss if you think that's better.

Reply:
Where is the parent block `input-field`?

Reply:
there's no parent, it's a sibling. in the old code it was handled with a +

Reply:
Oh I see, would it be less coupled if we added a mixin for this common icon style instead?

Reply:
sure, I will do that instead.